development_utilities/get_all_messages.py

```diff
@@ -0,0 +1,12 @@
+import inspect
```

Comment:
Is it correct that this suggestion is purely a refactoring request?

Since I'll be taking over this PR on behalf of @victorgarcia98, and I'm still a little uncertain as to increase the scope of this PR's code changes, would it be okay to tackle this refactoring in a separate PR?

Reply:
That is not an issue at all, lets do it in a seperate PR. It is an idea purely for refactoring/code deduplication. If you are busy, I can submit the PR for review to you.